packages/components/src/components/guild-navbar.tsx

```diff
@@ -62,9 +62,9 @@ export function getNavbarLogo(
   websiteName: string,
   description: string,
 ) {
-  return logo === HivePlatformLogo ? (
+  return logo === HiveCombinationMark ? (
     <Anchor title={websiteName} href="/">
-      <HivePlatformLogo />
+      <HiveCombinationMark />
```

Comment:
```suggestion
      < HivePlatformLogo />
```

@dotansimha wdy, I think is more beauty to remove `title` and `description` to save space in header  and use only title from svg like in screenshot https://guild-oss.slack.com/archives/C07DYCW4NUD/p1722521603235669

Reply:
I renamed the "PlatfromLogo" to "CombinationMark" to avoid confusion, because there's also a logo symbol in icons.